lib/SIL/Verifier/SILVerifier.cpp

```diff
@@ -1116,17 +1116,9 @@ class SILVerifier : public SILVerifierBase<SILVerifier> {
 
       if (VarInfo)
         if (unsigned ArgNo = VarInfo->ArgNo) {
-          // It is a function argument.
-          if (ArgNo < DebugVars.size() && !DebugVars[ArgNo].empty() && !VarInfo->Name.empty()) {
-            require(
-                DebugVars[ArgNo] == VarInfo->Name,
-                "Scope contains conflicting debug variables for one function "
-                "argument");
-          } else {
-            // Reserve enough space.
-            while (DebugVars.size() <= ArgNo) {
-              DebugVars.push_back(StringRef());
-            }
+          // Reserve enough space.
+          while (DebugVars.size() <= ArgNo) {
```

Comment:
Seems like you could express this as a conditional/std::max-ified `DebugVars.resize()`?